testcases/TC06.2-all-pass-1.md

```diff
@@ -0,0 +1,60 @@
+title: Decorative Image - decorative image on webpage
+
+
+# The Test Case ID should follow the pattern:
+# TC[Baseline Test Procedure #]-[Test Instruction #]-
+# [Expected Result (pass/fail/dna)]-[example #], e.g., TC05.1-1-fail-1
+tcid: TC06.2-pass-1
+
+# Description of the Test Case, the included code sample, test considerations,
+# and rationale for the expected result according to the applicable ICT
+# Baseline test
+descr: |
+    Identify any decorative image that is pure decoration, is used only for visual formatting, or is not presented to users. Check if the aria role is "none", "presentation", the aria-hidden is set to true,
+    the text alternative is empty, or if the image is inserted with CSS.
+
+    The code sample provides an equivalent description of the image in a <code>role = "presentation"</code> attribute. Assistive Technologies should ignore the presence of the image, which is why a successful test should identify a PASS against Baseline 6.2 Decorative Images.
+
+# Reference and link to the applicable ICT Baseline test
+app-baseline: |
+    [6.2 Test Procedure for Meaningful Images](https://section508coordinators.github.io/ICTTestingBaseline/06Images.html#62-test-procedure-for-decorative-images)
+
+    **Baseline Test ID:** 6.2-DecorativeImage
+
+    **Test Instruction:** All
+
+# Expected result that the ICT Baseline would predict
+# [Pass | Fail | DNA]
+result: PASS
+
+# Brief description of the rationale for the expected result
+result-descr: The image in the code sample is a decorative image from its <code>role="presentation"</code> attribute.
+
+# URL for the code sample
+# In the sample code file, add id="tc_code" to the
+# element that contains the relevant code snippet.
+#
+# Then upload the code sample to the 'testfiles' folder
+# and provide the link (and only the url) below.
+sample: /testfiles/TF06/06.2-all-pass-1.html
+
+# Table of test instructions, including the following table headers:
+# Test Instruction #; Instruction Detail; Expected Test Case Result
+#
+# Include the table in the content section below
+---
+| Test Instruction | Instruction Detail | Expected Test Case Result |
+|------------------|--------------------|---------------------------|
+| **IC-1** | Identify any image that conveys decoration | One `<img>` exists in the test data code sample. The image is meaningful and conveys information, based on the text directly below the image. |
+
+| **6.2-1** | Descriptions of the image that are provided by page content must be programmatically associated. | Not applicable |
+| **6.2-2** | Check to see if the image was inserted with CSS | Pass |
+| **6.2-3** | Check that the ARIA role is "presentation".| Pass |
+| **6.2-4** | Check that the ARIA role is "none". | Pass |
+| **6.2-5** | Check that aria-hidden is set to "true". | Pass |
+| **Result** | If any of the above checks fail, then Baseline Test 6.2-DecorativeImage fails. | All checks PASS (or do not apply). |
```

Comment:
the 6.2 Results is "If all of the above checks fail, then Baseline Test 6.2-DecorativeImage fails." 

Since the first check passed, this result is Pass.